onecore_maintenance_extension/models/maintenance.py

```diff
@@ -90,6 +89,26 @@ class OneCoreMaintenanceRequest(models.Model):
 
     # New fields for the form view only (not stored in the database)
     today_date = fields.Date(string='Today', compute='_compute_today_date', store=False)
+    new_mimer_notification = fields.Boolean(
+        string='New Mimer Message',
+        compute='_compute_new_mimer_notification',
+        store=False,
+    )
+    # This functions searches for notifications from Mimer.nu that are unread for the logged in user.
+    @api.depends('message_ids.notification_ids.is_read', 'message_ids.notification_ids.notification_type')
+    def _compute_new_mimer_notification(self):
+        for record in self:
+            message_ids = record.message_ids.ids
+
+            unread_mimer_notifications = self.env['mail.notification'].search([
+                ('mail_message_id', 'in', message_ids),
+                ('res_partner_id', '=', self.env.user.partner_id.id),
+                ('is_read', '=', False),
+                ('notification_type', '=', 'inbox'),
+                ('mail_message_id.author_id.user_ids.login', '=', 'odoo@mimer.nu')
```

Comment:
we should make sure that we have this user in all env:s. Can we add it via some hook?

Reply:
New story maybe?

Reply:
Yes!